src/tests/multihost/alltests/test_kcm.py

```diff
@@ -52,3 +52,36 @@ def test_client_timeout(self, multihost, backupsssdconf):
                                                    " /var/log/sssd/"
                                                    "sssd_kcm.log")
         assert 'Terminated client' in grep_cmd.stdout_text
+
+    def test_kcm_check_socket_path(self, multihost, enable_kcm):
+        """
+        @Title: kcm: Test socket path when sssd-kcm is activated by systemd
+        #https://github.com/SSSD/sssd/issues/5406
+        """
+        # Start from a known-good state after removing log file and adding a
+        # new socket path
+        multihost.master[0].service_sssd('stop')
```

Comment:
The tests in basic/* are run on a single vm, master and client are the same, so this would work if the test was added under basic. In this module test will run on a client. so we should replace master here with client. We can start with
```
multihost.client[0].service_sssd('stop')
tools = sssdTools(multihost.client[0]) 
```

Reply:
I wonder, what's the difference between basic and alltests folders? Is the only difference that the tests are executed on a single VM?